### DISCUSSION TITLE
无法验证邮箱

### DISCUSSION THREAD

**Discussion by Goojoe** [geeknote-net/feedback#22] [Q&A]
邮箱是Gmail
浏览器：Chromium 105.0.5128.0 64位
![image](https://user-images.githubusercontent.com/78355492/181006130-604991c9-cf90-4402-8b96-6bd9705cfef3.png)

**Comment by chloerei** [comment]
多谢，是 bug，已经修复了，可以再点同一个链接试试。

**Reply by Goojoe** [reply]
已成功，谢谢
